packages/botonic-core/src/handoff.js

```diff
@@ -140,9 +140,9 @@ export async function getAvailableAgentsByQueue(session, queueId) {
 }
 
 export async function getAvailableAgents(session) {
-  let baseUrl = session._hubtype_api || 'https://api.hubtype.com'
-  let botId = session.bot.id
-  let resp = await axios({
+  const baseUrl = session._hubtype_api || 'https://api.hubtype.com'
+  const botId = session.bot.id
+  const resp = await axios({
```

Comment:
"let"ophobia? 😆 is it very wrong to use `let` in all the pieces of code of this commit?

Reply:
:-D
I'm not too strong on this one
some people are not willing to pay the 2 extra keystrokes, and I respect that
But immutability eases your mind and side-effects are a bitch, so I left it since it's in the eslint recommended list

Reply:
hahaha nice!! 🎉